package.json

```diff
@@ -30,8 +30,8 @@
   "homepage": "https://github.com/starkbank/ecdsa-node#readme",
   "dependencies": {
     "axios": "^1.6.2",
-    "core-node": "^0.0.1",
-    "starkbank-ecdsa": "^1.1.5"
+    "starkbank-ecdsa": "^1.1.5",
+    "starkcore": "^0.0.2"
```

Comment:
tem como alterar nome de lib no npm ?? tipo, o nome da lib antes tava como "core-node", dai vc mudou pra "starkcore" ou vc subiu como ou lib diferente ?? pq se for outra lib a versão dele vai ser "0.0.1"

Reply:
Não, o Dotts mudou tudo e eu só mandei baixar o pacote e ele colocou a dependencia sozinha